packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1112,6 +1166,10 @@ class _DropdownButtonState<T> extends State<DropdownButton<T>> with WidgetsBindi
     Widget result = DefaultTextStyle(
       style: _textStyle,
       child: Container(
+        decoration: BoxDecoration(
+          color:_hasPrimaryFocus ? widget.focusColor ?? Theme.of(context).focusColor : null,
+          borderRadius: const BorderRadius.all(Radius.circular(4.0)),
```

Comment:
Is this addition of a border radius intentional? Is there something about the focus changes that needs a rounded rect? Or did this get moved from somewhere else that I missed?

Reply:
Yes, it's intentional: there's no spec for this, so I'm emulating the other defaults for highlights.  It didn't get moved from anywhere because it didn't exist before, since we couldn't focus this component.